judge/utils/tests/test_camo.py

```diff
@@ -0,0 +1,92 @@
+import hmac
+from hashlib import sha1
+
+from django.test import SimpleTestCase
+from lxml.html import HTMLParser, fromstring, tostring
+
+from judge.utils.camo import CamoClient
+from judge.utils.unicode import utf8bytes
+
+
+# TODO: Test the setting of judge.utils.camo.client depending on settings variables. I've tried my hardest... but the
```

Comment:
I would be very surprised if https://docs.djangoproject.com/en/2.2/topics/testing/tools/#overriding-settings doesn't work.

Reply:
I've tried it and the other options I could find to. Perhaps it has something to do with how python modules are loaded in general, I'm not really sure.

I'll try again though, but overall this probably isn't too important considering it would just check whether a client gets set or not.